pkg/efi/manager.go

```diff
@@ -0,0 +1,198 @@
+// nolint:goheader
```

Comment:
I supposed that the linter is disabled here because it's not a file created by SUSE for this project, right? In that case maybe just small comment to explain this could be helpful no?

But I agree that with the `Copyright 2021 Canonical Ltd.` below perhaps it's enough?

Reply:
Good points @ldevulder! I will try to change it! :+1: